sdk/eventhub/azure-eventhub/azure/eventhub/_consumer_client.py

```diff
@@ -58,8 +58,14 @@ class EventHubConsumerClient(ClientBase):
     :keyword float auth_timeout: The time in seconds to wait for a token to be authorized by the service.
      The default value is 60 seconds. If set to 0, no timeout will be enforced from the client.
     :keyword str user_agent: The user agent that should be appended to the built-in user agent string.
-    :keyword int retry_total: The total number of attempts to redo a failed operation when an error occurs. Default
-     value is 3.
+    :keyword int retry_total: The total number of attempts to redo a failed operation when an error occurs.
+     Default value is 3. The context of `retry_total` in receiving is special: The `receive` method is implemented
+     by a while-loop calling internal receive method in each iteration. In the `receive` case,
+     `retry_total` specifies the numbers of retry after error raised by internal receive method in the while-loop.
+     If retry attempts exhaust, the `on_error` callback would be called if provided carrying the error information,
```

Comment:
"attempts are exhausted"
"would be called (ifprovided) with the error information. The failed internal partition consumer will be closed..."
"and a new internal partition consumer will be created...."

Reply:
This comment also applies to doc change on line 174

Reply:
thanks, updated